src/ui/server/controllers/insurance/policy-and-export/map-submitted-data/policy-contact/index.test.ts

```diff
@@ -47,24 +47,9 @@ describe('controllers/insurance/policy-and-export/map-submitted-data/policy-cont
 
       const expected = {
         [IS_SAME_AS_OWNER]: false,
-      };
-
-      expect(result).toEqual(expected);
-    });
-  });
-
-  describe(`when ${NAME} is ${OTHER_NAME} and ${POSITION} is provided`, () => {
-    const mockBody = {
-      [NAME]: OTHER_NAME,
-    };
-
-    it(`should return an object without ${NAME} and add ${IS_SAME_AS_OWNER} as false and set ${POSITION} to empty string`, () => {
-      mockBody[POSITION] = 'CEO';
```

Comment:
Is this test / logic no longer relevant?

Reply:
no, so the new test I added below and the modified one above are needed.  No idea what I was doing with this test